crates/sui-core/src/authority.rs

```diff
@@ -519,11 +571,20 @@ impl AuthorityState {
         // to do this, since the false contention can be made arbitrarily low (no cost for 1.0 -
         // epsilon of txes) while solutions without false contention have slightly higher cost
         // for every tx.
-        let tx_guard = self.database.acquire_tx_guard(&certificate).await?;
+        let span = tracing::debug_span!(
```

Comment:
You could just use the `#[instrument()]` macro on the acquire_tx_guard function itself.

Reply:
@velvia what would be the name of the prometheus histogram if we used `#[instrument()]`?

Reply:
`tracing_span_latencies` ... see
https://github.com/MystenLabs/mysten-infra/blob/main/crates/telemetry-subscribers/src/span_latency_prom.rs